src/app/components/Billboard/index.styles.ts

```diff
@@ -0,0 +1,99 @@
+import { css, Theme } from '@emotion/react';
+import pixelsToRem from '../../utilities/pixelsToRem';
+
+export default {
+  link: ({ palette }: Theme) =>
+    css({
+      textDecoration: 'none',
+      '&:hover, &:focus': {
+        color: palette.WHITE,
+        textDecoration: 'underline',
+      },
+    }),
+  headerContainer: ({ mq }: Theme) =>
+    css({
+      display: 'flex',
+      flexDirection: 'column',
+      position: 'relative',
+      [mq.HIGH_CONTRAST]: {
+        borderBottom: `solid ${pixelsToRem(1)}rem transparent`,
+      },
+    }),
+  backgroundContainer: ({ palette }: Theme) =>
+    css({
+      position: 'absolute',
+      top: '0',
+      backgroundColor: palette.GREY_10,
+      bottom: '0',
+      width: '100%',
+      overflow: 'hidden',
+    }),
+  contentContainer: ({ mq, gridWidths }: Theme) =>
+    css({
+      [mq.GROUP_4_MIN_WIDTH]: {
+        maxWidth: `${pixelsToRem(gridWidths[1280])}rem`,
+        margin: '0 auto',
+        position: 'relative',
+        width: '100%',
+      },
+    }),
+  heading: ({ palette }: Theme) =>
+    css({
+      paddingTop: '1.5rem',
+      paddingBottom: '0.5rem',
+      color: palette.WHITE,
+    }),
+  liveLabelPulse: ({ mq, palette, spacings }: Theme) =>
+    css({
+      width: `${spacings.HALF + spacings.DOUBLE}rem`,
+      height: `${spacings.HALF + spacings.DOUBLE}rem`,
+      color: palette.LIVE_LIGHT,
+      [mq.GROUP_1_MIN_WIDTH]: {
+        width: `${spacings.TRIPLE}rem`,
+        height: `${spacings.TRIPLE}rem`,
+      },
+      [mq.GROUP_3_MIN_WIDTH]: {
+        width: `${spacings.TRIPLE + spacings.HALF}rem`,
+        height: `${spacings.TRIPLE + spacings.HALF}rem`,
+      },
+      [mq.HIGH_CONTRAST]: {
+        color: 'canvasText',
+      },
+    }),
+  liveLabelText: ({ palette }: Theme) =>
+    css({
+      'span:first-of-type': {
+        color: palette.LIVE_LIGHT,
+      },
+    }),
+  textContainerWithImage: ({ mq, spacings }: Theme) =>
+    css({
+      position: 'relative',
+      padding: `${spacings.FULL}rem ${spacings.FULL}rem ${spacings.DOUBLE}rem`,
+      [mq.GROUP_2_MIN_WIDTH]: {
+        padding: `${spacings.FULL}rem ${spacings.DOUBLE}rem ${spacings.DOUBLE}rem`,
+      },
+      [mq.GROUP_4_MIN_WIDTH]: {
+        padding: `${spacings.DOUBLE}rem`,
+        minHeight: `${pixelsToRem(440)}rem`, // calculation includes padding
+        height: '100%',
+        display: 'flex',
+        flexDirection: 'column',
+        justifyContent: 'center',
+        maxWidth: '50%', // determines width of text overlay.
+      },
+    }),
+  titleWithImage: ({ palette }: Theme) =>
```

Comment:
As above, we could rename this to remove mention of image

```suggestion
  title: ({ palette }: Theme) =>
```